### DISCUSSION TITLE
How to add compressive stress on dendrite growth surface？

### DISCUSSION THREAD

**Discussion by biaogxb** [idaholab/moose#21547] [Q&A Modules: Phase field] [answered]
I want to study the growth of dendrites by applying stress on the surface. How to add compressive stress on dendrite growth surface？
As shown in the picture below：

![image](https://user-images.githubusercontent.com/95063954/178205924-23b5ad68-a099-4f73-86e5-e3c77d53c098.png)
[Zhang_2021_J._Electrochem._Soc._168_090522.pdf](https://github.com/idaholab/moose/files/9081546/Zhang_2021_J._Electrochem._Soc._168_090522.pdf)
I studied some examples. Is this an example of increasing compressive stress on the surface？https://github.com/idaholab/moose/blob/next/modules/combined/examples/phase_field-mechanics/interface_stress.i

**Comment by GiudGiud** [comment, marked as answer]
@dschwen @recuero who can I tag on dendrites?
Are Zijian Hong and Venkatasubramanian Viswanathan on github and active on the MOOSE repo?

**Reply by recuero** [reply]
@SudiptaBiswas?

**Reply by SudiptaBiswas** [reply]
@biaogxb I don't think we have the exact example you are looking for. This can be done with the phase-field mechanics approach. You will have to implement the free energy components though.
